src/game/client/water/w3dwater.cpp

```diff
@@ -1069,7 +1072,7 @@ bool WaterRenderObjClass::Get_Clipped_Water_Plane(CameraClass *cam, AABoxClass *
 
 inline unsigned long F2DW(float f)
 {
-    return *((unsigned *)&f);
+    return *(reinterpret_cast<unsigned long *>(&f));

```

Comment:
uint32_t suggest also changing the function signature to uint32_t

Reply:
Yeah good point. There is another review with a 2 more of these functions. I think that should be done in a new review entirely.